lib/main_common.pm

```diff
@@ -207,6 +208,12 @@ sub rescuecdstep_is_applicable {
     return get_var("RESCUECD");
 }
 
+sub bootencryptstep_is_applicable {
+    # activating an existing encrypted volume but not forcing recompute does
+    # *not* yield an encrypted system, see bsc#993247 fate#321208
+    return get_var('ENCRYPT') && !(get_var('ENCRYPT_ACTIVATE_EXISTING' && !get_var('ENCRYPT_FORCE_RECOMPUTE')));
```

Comment:
there was a missing tick `'` behind 'ENCRYPT_ACTIVATE_EXISTING', maybe this confused?